subjects/cybersecurity/hole-in-bin/README.md

```diff
@@ -1,26 +1,44 @@
-# Hole in Bin
+# Hole-In-Bin
+
+<center>
```

Comment:
This is the only one with the meme maybe its not consistent but i leave it up to you to decide if you leave it or remove it.

Reply:
I will keep it, It will add a taste to the project